tests/lcs/core/engine/send.livecodescript

```diff
@@ -33,6 +33,27 @@ on TestSendParams
 	TestAssert "send script with multiple params", the result is "2"
 end TestSendParams
 
+/*
+It is known that there is code in the wild relying on the following 
+(undocumented) form of the send command, so we add a test here to ensure 
+backwards compatibility until such a point as it is deemed possible or 
+necessary to break.
+*/
+on TestSendLegacy
+	local tStack
+	create stack
+	put it into tStack
+	set the defaultStack to the short name of tStack
+	create button
+	
+	local tSendScript
+	put "delete button 1 of me" into tSendScript
```

Comment:
@livecodeali should we have a test that does parameter evaluation in the calling script context?